src/components/mission/Mission.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Spinner } from 'react-spinners-css';
+import style from './Mission.module.css';
+import { displayMissionFunction, missionStatusFunction } from '../../redux/mission/Mission';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+const Mission = () => {
+  // const [isMember, setMember] = useState(false);
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 👍🏻